calblueprint/siren
# Tracker

# ✨ New in this PR ✨

## Added tracker component🚨 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Researched different tracker packages
 - [x] Added tracker implementation in sync with Firestore
 - [x] Styled tracker
 - [ ] Add minimizing button to tracker

## How can the reviewer test your code? 👩‍💻
Run 'expo start' and log into the application. Navigate to the home page.

The layout should look like the following:
![IMG_0220](https://user-images.githubusercontent.com/57604304/143974121-8cde7ff9-8798-4198-95ba-cc493469d46f.jpeg)



## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 
*Still need to finish styling and need to remove comments. Since only one client has cases, the userId is hard coded in.

## Resources 📔
_link online resources and related PRs_ 
https://www.npmjs.com/package/react-native-progress-steps

🧜‍♀️ cc: @shannonbonet @phoebeli23
